saq/queue/postgres.py

```diff
@@ -107,6 +110,7 @@ def __init__(
         self.versions_table = Identifier(versions_table)
         self.jobs_table = Identifier(jobs_table)
         self.stats_table = Identifier(stats_table)
+        self.metadata_table = Identifier(metadata_table)
```

Comment:
this should go into the stats table now that we have migrations

Reply:
I've pushed a branch to the PR in which metadata sits in the same table. 

That said, I am not 100% sure about it. 

We get a small win in performance in avoiding a join, but I don't like the `ttl` behavior. It doesn't quite behave the same way as Redis because you have to share the TTL for metadata and stats. 

My best suggestion would be to merge stats and metadata to avoid this weird behavior. WDYT?

Reply:
what do you mean merge?

Reply:
I mean we merge the code top to bottom, get rid of the two separate jobs/calls/methods and just have one stats/info method both for the retrieving of data (which we already do) but also for the pushing of data.

Reply:
sure sounds good